docs/index.md

```diff
@@ -6,32 +6,35 @@ Internally, Agent Evaluation implements an LLM agent ([evaluator](evaluators/ind
 
 ## Key features
 
-- Built-in support for popular AWS services including [Amazon Bedrock](https://aws.amazon.com/bedrock/), [Amazon Q Business](https://aws.amazon.com/q/business/), and [Amazon SageMaker](https://aws.amazon.com/sagemaker/). You can also [bring your own agent](targets/custom_targets.md) to test using Agent Evaluation.
-- Orchestrate concurrent, multi-turn conversations with your agent while evaluating its responses.
-- Define [hooks](hooks.md) to perform additional tasks such as integration testing.
-- Can be incorporated into CI/CD pipelines to expedite the time to delivery while maintaining the stability of agents in production environments.
+✅ Evaluate an agent's responses by simulating concurrent, multi-turn conversations.
+
+✅ Built-in support for popular AWS services including [Amazon Bedrock](https://aws.amazon.com/bedrock/), [Amazon Q Business](https://aws.amazon.com/q/business/), and [Amazon SageMaker](https://aws.amazon.com/sagemaker/). You can also [bring your own agent](targets/custom_targets.md) to test using Agent Evaluation.
+
+✅ Define [hooks](hooks.md) to perform additional tasks such as integration testing.
+
+✅ Incorporate into CI/CD pipelines to expedite the time to delivery while maintaining the stability of agents in production environments.
 
 ---
 
 <div class="grid cards" markdown>
 
--   :material-rocket-launch-outline:{ .lg .middle } __Getting started__
+-   🚀 __Getting started__
```

Comment:
Not related to this PR. Can we also change the src/agenteval/templates/summary/agenteval_summary.md.jinja to use 🟢🔴 instead of short codes?

Reply:
Good idea!